user-preferences-explainer.md

```diff
@@ -63,6 +63,12 @@ In addition to the preferences `color_scheme_dark` and `color_scheme_light`, oth
 
 There are minimal security and privacy concerns with this proposal. This will allow sites to know what user preferences are set and use this for fingerprinting, however this information is already exposed through CSS.
 
+## Alternatives considered
+
+Use the CSS media query syntax for the keys (e.g. `(prefers-color-scheme: dark)`) and parse this as CSS. However, using the CSS parser adds a lot of complexity which we would like to avoid during manifest parsing.
```

Comment:
You could phrase this a little more like an alternative, eg. "It would also be possible to use [...], instead of the fixed keys as proposed".
Maybe also expand on the complexity (which also happens during window opening and theoretically constantly), eg. "Before a web app window can launch, a CSS parser would need to be run to analyse the media query" and maybe give an example of a bad situation that is hard and not necessary, like a media query based on window size.